setup.py

```diff
@@ -36,7 +36,7 @@
     long_description_content_type="text/markdown",
     url='http://github.com/silvairgit/python-bluetooth-mesh',
     packages=find_packages(exclude=('test*', )),
-    python_requires='>=3.6.0',
+    python_requires='>=3.6.0,<3.8.0',
```

Comment:
Why we don't support Python 3.8?

Reply:
I ran into some issues with `construct` and don't have time to investigate this now. Feel free to raise an issue.